src/models/BlogTag.ts

```diff
@@ -0,0 +1,27 @@
+import { DataTypes, Model } from "sequelize";
+import sequelize from "../config/databaseConnection";
+
+class BlogTag extends Model {
+  declare id: number;
+  name: string;
+}
+
+BlogTag.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    sequelize,
+    tableName: "blogTags",
```

Comment:
Instead of use camelCase for table names and table rows we should use snake case to comply with postgres conventions.